ccm_web/server/src/invitation/cirrus-invitation.interfaces.ts

```diff
@@ -0,0 +1,25 @@
+import { hasKeys } from '../typeUtils'
+
+export const cirrusAPIVersion = '/api/v1'
+export const CirrusAPIEndPoint = '/batchInviteCsv'
```

Comment:
This should be lowercase `c`. Or if you prefer, you could do all caps with underscores for constants, e.g. `CIRRUS_API_ENDPOINT`. The `CamelCase` implies it's a class or type/interface in TS.
```suggestion
export const cirrusAPIEndPoint = '/batchInviteCsv'
```

Reply:
Sure!  I didn't notice that.  Looks like I took that straight from @pushyamig's suggested code.

Reply:
Oh, don't blame it on @pushyamig. You can blame it on me if you'd like for only suggesting the change here.